tests/bankHome/homePageContainsLoginButtons.spec.js

```diff
@@ -0,0 +1,14 @@
+import { test } from '@playwright/test';
+import { BankHomePage } from '../../src/pages/BankHomePage';
+
+
+test('Assert homePage contains Login buttons', async ({ page }) => {
+
+const bankHomePage = new BankHomePage(page); 
+
+
+await bankHomePage.open();
+await bankHomePage.assertCustomerLoginButtonIsVisible();
+await bankHomePage.assertBankManagerLoginButtonnIsVisible();
```

Comment:
There is a typo in the method name `assertBankManagerLoginButtonnIsVisible`. It should be corrected to `assertBankManagerLoginButtonIsVisible` to match the method defined in the `BankHomePage` class.